eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/service/CommentService.java

```diff
@@ -0,0 +1,49 @@
+package hu.flowacademy.companycalendar.service;
+
+import hu.flowacademy.companycalendar.model.Comment;
+import hu.flowacademy.companycalendar.model.dto.CommentDTO;
+import hu.flowacademy.companycalendar.repository.CommentRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+@Service
+@Transactional
+@AllArgsConstructor
+public class CommentService {
```

Comment:
A függvények várjanak paraméterként Comment entity-t és azt is adjon vissza. Minden DTO a controller-ben legyen átalakítva entitássá és aztán a response vissza.